### PR TITLE
Cadc 12493 Updated skaha to support an image with multiple types/labels

### PR DESCRIPTION
**Pull request by yeunga**
Here’s an excerpt of the output from the following curl command to list the images. The output shows images with more than one type/label.
curl -v -E ~/.ssl/cadcproxy.pem https://rc-uv.canfar.net/skaha/image
…
  {
    "id": "images.canfar.net/uvickbos/pycharm:0.1",
    "types": [
      "headless",
      "desktop-app"
    ],
    "digest": "sha256:203dde66cb2a4d3c4084c4439ae15499f0996961f203d6c0a29c8a65e0421572"
  },
  {
    "id": "images.canfar.net/uvickbos/swarp:0.1",
    "types": [
      "headless",
      "desktop-app"
    ],
    "digest": "sha256:af919892a3ce3b46729f5fe5815a643a30298da39f03a2086620752c78b1b880"
  },
  {
    "id": "images.canfar.net/uvickbos/iraf:0.2",
    "types": [
      "desktop-app"
    ],
    "digest": "sha256:09595ea5b2212f2438232835aafd126efa30232629e81d49af02fe96d64ff644"
  },
  {
    "id": "images.canfar.net/uvickbos/aram:0.1",
    "types": [
      "notebook"
    ],
    "digest": "sha256:1593dea69567b0a53337713db68246b3f1d9cf94147cdfa8d418eb0b8cf7a7c8"
  },

### CODE REVIEW

File: deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
           value: "ivo://cadc.nrc.ca/gms?skaha-users"
         - name: skaha.adminsgroup
           value: "ivo://cadc.nrc.ca/gms?skaha-admins"
-        image: images.canfar.net/skaha-system/skaha:0.9.21
+        image: images.canfar.net/skaha-system/skaha:0.9.22
```
Comment (brianmajor) [inline deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml]: This PR causes an API change.  Let's make it version 0.10.0

**Review by brianmajor — CHANGES_REQUESTED** [review]
Looks good Alinga--just the version comment.

File: deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml (hunk 1 above)
Comment (yeunga) [reply deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml]: Good point!

File: skaha/src/main/webapp/service.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 swagger: '2.0'
 info:
-  version: 0.3
+  version: 0.4
```
Comment (brianmajor) [inline skaha/src/main/webapp/service.yaml]: I guess this should match the version of skaha, 0.10.0?

**Review by brianmajor — CHANGES_REQUESTED** [review]

File: skaha/src/main/webapp/service.yaml (hunk 2 above)
Comment (yeunga) [reply skaha/src/main/webapp/service.yaml]: Okay.

**Review by brianmajor — DISMISSED** [review]

**Comment by brianmajor** [comment]
Will merge this once we know that the portal is happy.

**Review by brianmajor — APPROVED** [review]

**Pull request merged**